engine/kv_engine.cpp

```diff
@@ -979,7 +979,7 @@ Status KVEngine::HashGetImpl(const StringView& key, std::string* value,
     void* pmem_record = nullptr;
     if (hash_entry.GetRecordType() == StringDataRecord) {
       if (TimeUtils::CheckIsExpired(
-              hash_entry.GetIndex().string_record->GetExpiredTime())) {
+              hash_entry.GetIndex().string_record->ExpireTime())) {
```

Comment:
Not need to change API, there are a lot of Getxxx() around, or we change them altogether later